src/content/enterprise/install/releases.mdx

```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.0.0
+September 1, 2022
+
+### Removal Notice
+- Kubernetes [1.19](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md) is no longer supprted.
+
+### Deprecation Notice
+- Support for Kubernetes [1.20](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md) has been derprecated and will be removed next release.
+
+### Features
+- License enforcements changed to allow for easier adoption.
+    - Unlicensed instances will be limited to 1 user.
+    - Instances with a trial license will be limted to 3 users.
+    - Scale and Enterprise limits will be softly enforced to ease user management and promote adoption of your installation.
```

Comment:
I don't think we need to mention this here.

Reply:
- [ ] Remove this.